app/record_translation.rb

```diff
@@ -0,0 +1,10 @@
+# frozen_string_literal: true
+
+module RecordTranslation
+  def self.deserialize_line(line)
+    throw 'Invalid Input' if !line.include?('|') && !line.include?(',')
```

Comment:
if you're trying to throw an error here, what you want it `raise`, not `throw`

https://stackoverflow.com/questions/51021/what-is-the-difference-between-raising-exceptions-vs-throwing-exceptions-in-ruby